terrain_ingester.py

```
@@ -0,0 +1,386 @@
+        print(f"Error: No .terrain files found in {dir_path}", file=sys.stderr)
+        sys.exit(1)
+    
+    print(f"Processing {len(terrain_files)} .terrain files...", file=sys.stderr)
+    
+    added = 0
+    skipped = 0
+    
+    for terrain_file in terrain_files:
+        try:
+            parsed = parse_terrain_file(str(terrain_file))
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)